src/augment/types/count.js

```diff
@@ -0,0 +1,20 @@
+import { GraphQLInt, GraphQLNonNull, GraphQLString } from 'graphql';
+import { buildNeo4jTypes } from './types';
+
+export const CountType = {
```

Comment:
Should we call this `AggregateType`? Or just `const CountType= 'Count'`?

Reply:
i think eventually this will also have `groupBy` so its planning ahead a bit

unsure on the Aggregate vs Count, since the aggregate (sum, min, avg, max) only exist for number fields to group by 🤔